setup.py

```diff
@@ -25,7 +22,6 @@
     entry_points={'console_scripts': ['pyxrf = pyxrf.gui:run']},
     package_data={'pyxrf.view': ['*.enaml'], 'configs': ['*.json']},
     include_package_data=True,
-    install_requires=['scikit-xray', 'matplotlib', 'enaml', 'six', 'numpy'],
```

Comment:
got it. thanks. So if using conda-recipe, we will remove install_requires most of the time.